__tests__/test-data.test.js

```diff
@@ -0,0 +1,53 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  db.end();
+});
+
+describe("api/topics", () => {
+  test("recieve status code 200", () => {
```

Comment:
What are we getting a status code of 200 for? I know its for topics, so do you, but could this be more descriptive?